thread/unix/omrthreadattr.c

```diff
@@ -288,7 +287,7 @@ omrthread_attr_set_priority(omrthread_attr_t *attr, omrthread_prio_t priority)
 		return J9THREAD_SUCCESS;
 	}
 
-	if (J9THREAD_VALUE_OUT_OF_RANGE(priority, J9THREAD_PRIORITY_MIN, J9THREAD_PRIORITY_MAX)) {
+	if (priority > J9THREAD_PRIORITY_MAX) {
```

Comment:
This now assumes `J9THREAD_PRIORITY_MIN` is zero; I suggest we don't want to do that.

Reply:
On the other hand, that assumption seems already baked into the code in other places like:
* https://github.com/eclipse/omr/blob/master/thread/aix/priority.c#L52
* https://github.com/eclipse/omr/blob/master/thread/common/omrthread.c#L2681

so changing `J9THREAD_PRIORITY_MIN` to something other than zero would be non-trivial.

Reply:
Are suggesting to use `J9THREAD_PRIORITY_MIN` instead of `0` in those two locations?

AND

Use logic of the below form for the validity check?
```
isValid(priority, J9THREAD_PRIORITY_MIN, J9THREAD_PRIORITY_MAX, UINTPTR_MIN, UINTPTR_MAX);
{
   if (J9THREAD_PRIORITY_MIN > UINTPTR_MIN) {
      if (priority < J9THREAD_PRIORITY_MIN) {
         return false;
      }
   }

   if (J9THREAD_PRIORITY_MAX < UINTPTR_MAX) {
      if (priority > J9THREAD_PRIORITY_MAX) {
         return false;
      }
   }

   return true;
}
```

Reply:
I don't expect the list of "other places" above is complete. I was retracting my objection to the (new) places that make that assumption.